apps/rock-paper-scissors/src/pages/home/create/Create.tsx

```diff
@@ -0,0 +1,71 @@
+import { Hex } from '@gear-js/api';
+import { useCreateHandler } from '@gear-js/react-hooks';
+import { Button, Input } from '@gear-js/ui';
+import { useForm } from '@mantine/form';
+import { BackButton } from 'components';
+import { isExists, isMinValue } from 'utils';
+import styles from './Create.module.scss';
+
+type Props = {
+  onBackClick: (arg: string) => void;
+  onSubmit: ReturnType<typeof useCreateHandler>;
+  setLoading: (loading: boolean) => void;
+  setStateAction: (hex: Hex) => void;
+};
+
+const initialValues = {
+  // gameName: '',
+  betSize: '',
+  playersCountLimit: '',
+  entryTimeoutMs: '',
+  moveTimeoutMs: '',
+  revealTimeoutMs: '',
+};
+const transformValues = (values: any) => ({
+  betSize: values.betSize,
+  playersCountLimit: values.playersCountLimit,
+  entryTimeoutMs: `${values.entryTimeoutMs}000`,
+  moveTimeoutMs: `${values.moveTimeoutMs}000`,
+  revealTimeoutMs: `${values.revealTimeoutMs}000`,
+})
+
+const validate = {
+  // gameName: isExists,
+  betSize: (value: string) => (isExists(value) || isMinValue(Number(value), 1000)),
```

Comment:
get rid of `isMinValue` check it's gonna be hard to maintain since that value depends on contract